### PR TITLE
Back button add on blogpost

### PR DESCRIPTION
**Pull request by waleed345**

### CODE REVIEW

File: src/Containers/BlogPost.js [hunk 1]
```diff
@@ -131,46 +124,55 @@ export default function BlogHome() {
...
     }
   }, [loading, error, data, setReactionFun]);
 
   if (loading) {
-    return <Loader />
+    return <Loader />;
   }
 
+  const onBackClick = () => {
+    window.location.replace("#/");
```
Comment (jvm-odoo) [inline src/Containers/BlogPost.js]:
```suggestion
    window.location.replace(document.referrer);
```
https://developer.mozilla.org/en-US/docs/Web/API/Document/referrer

**Comment by saadpasta** [comment]
@waleed345 please try to maintain the indentation of the code.

**Comment by waleed345** [comment]
Screen Shot 2020-03-09 at 11 31 29 AM

**Comment by saadpasta** [comment]
@allcontributors please add @waleed345 for code

**Pull request merged**